### PR TITLE
HWS: Support table default miss action parsing

### PR DESCRIPTION
**Pull request by Itamar-Gozlan**
Add support for displaying a new feature that
defines an HWS table as another HWS table default miss. The attribute will be presented if a default miss is defined.

### CODE REVIEW

File: hws/src/dr_common.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
 Returns the output string.
 """
 def dump_obj_str(keys, data):
+    # import pdb; pdb.set_trace()
```
Comment (hamdanigbaria) [inline hws/src/dr_common.py:46]: please remove

File: hws/src/dr_table.py [hunk 2]
```diff
@@ -101,3 +105,4 @@ def save_to_db(self):
             _db._term_dest_db[self.data.get("local_rx_icm_addr")] = {"type": "FT", "id": self.data.get("local_ft_id")}
         if self.data.get("local_tx_icm_addr") != "0x0":
             _db._term_dest_db[self.data.get("local_tx_icm_addr")] = {"type": "FT", "id": self.data.get("local_ft_id")}
+
```
Comment (hamdanigbaria) [inline hws/src/dr_table.py:108]: please remove

File: hws/src/dr_common.py (hunk 1 above)
Comment (Itamar-Gozlan) [reply hws/src/dr_common.py]: done

File: hws/src/dr_table.py (hunk 2 above)
Comment (Itamar-Gozlan) [reply hws/src/dr_table.py]: done

**Review by hamdanigbaria — APPROVED** [review]

**Pull request merged**